Story/DunkanAndFerzana[MEM].cs

```diff
@@ -0,0 +1,64 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class DunkanAndFerzana
```

Comment:
Npc naming is not descriptive as i told u so if lair doesn't have any other storyquest and the folder doesn't contain lair.cs file create it with that name lair[member]